### PR TITLE
chore(workflow-v2.0.0): releasing workflow-v2.0.0

### PR DESCRIPTION
**Pull request by bacongobbler**
It's the big one! :tada:

### CODE REVIEW

File: router-v2.0.0/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: router-v2.0.0
+home: https://github.com/deis/router
+version: v2-beta
```
Comment (sgoings) [inline router-v2.0.0/Chart.yaml]: This needs to change to v2.0.0

**Comment by bacongobbler** [comment]
chart has been updated to reference official images. Just waiting on those to be published before doing a sanity check.

**Comment by bacongobbler** [comment]
sanity check complete:

```
><> kd get po
NAME                          READY     STATUS    RESTARTS   AGE
deis-builder-mmsdf            1/1       Running   3          4m
deis-controller-5bqwb         1/1       Running   1          4m
deis-database-n6a1f           1/1       Running   0          4m
deis-logger-feodf             1/1       Running   0          4m
deis-logger-fluentd-om739     1/1       Running   0          3m
deis-minio-bvw5a              1/1       Running   0          4m
deis-monitor-grafana-1doxz    1/1       Running   0          4m
deis-monitor-influxdb-otah0   1/1       Running   0          4m
deis-monitor-stdout-uzri5     1/1       Running   0          4m
deis-monitor-telegraf-ev2lp   1/1       Running   0          3m
deis-registry-1oky6           1/1       Running   1          4m
deis-router-icbc9             1/1       Running   0          4m
deis-workflow-manager-htcd4   1/1       Running   0          4m
><> deis init
Registered bacongobbler
Logged in as bacongobbler
Uploading id_rsa.pub to deis... done
```

File: workflow-v2.0.0/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Workflow v2.0.0
+
+Please report any issues you find in testing Workflow to the appropriate GitHub repository:
```
Comment (vdice) [inline workflow-v2.0.0/README.md]: Should we change `testing Workflow` to `running Workflow` -- or a different verb other than `testing`?

Comment (bacongobbler) [inline workflow-v2.0.0/README.md]: good point. I'll make the change to "find while running Workflow"

File: router-v2.0.0/manifests/deis-router-dhparam-secret.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: deis-router-dhparam
+  namespace: deis
+  labels:
+    heritage: deis
+type: Opaque
+data:
+  dhparam: LS0tLS1CRUdJTiBESCBQQVJBTUVURVJTLS0tLS0KTUlJQkNBS0NBUUVBNGJPMlpCOTRYVVh3YjlSL0QxM0xFZ2d6V1U2Y0hZSXJzakdseVhUSkZHZmdsaFl5TUtnUQpKRXFsTjd5RWs0UUNVV25iTEtDU2F4anZCTVNqbWlTR3VvajNWRDlYRTVDaGx2SzlGQVp6alFGMFZ2VzdkNDJmCnNkNmlmcUYwSUV0R2lxWDRpTzhtWmhYZVJrc3BYekZXYkhyTklaanBCdmc5VDhmczY0ZCtDU2l2SXBqWWRVQ1UKQWRhNTViRDdLeWJkTWdJMFZza21jL3RpclN5SkIwZDVmUnYraG1vMW9pOG92Rm9vUFp2cS9HWGxPOHlRRzRZTgppWXdWYmU5c29wQ3lYNzEzaUtlM0pBQ29JWnUxZXgrRjVhbU53U3E3Rk9xb2JqK3Z2cURRM0Q2WEJaanczT3NuClFRWmVhTWYyMDVwOERWbFpWLzRqQUtQeEpUQkxyejh6c3dJQkFnPT0KLS0tLS1FTkQgREggUEFSQU1FVEVSUy0tLS0tCg==
```
Comment (arschles) [inline router-v2.0.0/manifests/deis-router-dhparam-secret.yaml:10]: (not blocking for this PR, just recording my thought here)

I wonder if we should add a comment here indicating that the user should generate their own `dhparam` and replace this one, and add a link to the docs.

Thoughts @krancour ?

**Pull request merged**